ci/test_python_cudf.sh

```diff
@@ -5,7 +5,7 @@
 cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../;
 
 # Common setup steps shared by Python test jobs
-source ./ci/test_python_common.sh
+source ./ci/test_python_common.sh test_python_cudf
```

Comment:
You'll need to make a key corresponding to this, or just rename the test_python key and remove the dask_cudf and cudf_pandas lists from that key.

Reply:
Done, could you take a look at the new changes? I had to retain `test_python` as that was being utilized by `ci/cudf_pandas_scripts/run_tests.sh`

Reply:
I would rename that to `test_python_cudf_pandas` and remove the unnecessary bits like polars to minimize the number of packages being installed into that environment.

Reply:
Is it okay if I push this particular change in a follow-up in the interest of unblocking the CI ASAP?

Reply:
Sure that is fine.

Reply:
CI is now passing and I incorporated the changes discussed here into this PR.